chatrooms.lua

```diff
@@ -29,12 +35,22 @@ function OnStartup()
 				sDescription = "Chatroom for usage by VIPs.",
 				sEmail = "donot@mail.me"
 			},
-			sFileName = "VIPChat.txt"
-		}
+			sFileName = "VIPChat.txt",
+			tChatHistory = {},
+			tUsers = {},
+		},
 	}
-	for sIndex, sValue in pairs(tChatRooms) do
-		Core.RegBot( sIndex, sValue.BOT.sDescription, sValue.BOT.sEmail, true )
+	for sIndex, tValue in pairs( tChatRooms ) do
+		Core.RegBot( sIndex, tValue.BOT.sDescription, tValue.BOT.sEmail, true )
 	end
+	dofile( sPath..sExtPath.."offliner.lua" )
```

Comment:
I used `offliner.lua` to use `GetModerators()` to follow the same thing with `!sm` [here] (https://github.com/HiT-Hi-FiT-Hai/ptokax-scripts/blob/master/botoffliner.lua#L133); and to get the whole list of mods instead of only the online ones, because even the mod who isn't online at that moment can still read the chats by `history` command. (Ideal much?) Although this fails in the case for `#[VIPChat]` and will only list the online VIPs because of [this] (https://github.com/nishithshah2211/ptokax-scripts/blob/chathistory/chatrooms.lua#L50).